config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1027,6 +1027,56 @@ presubmits:
         env:
         - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
           value: "1"
+  - name: pull-kubernetes-node-crio-cgrpv2-imagefs-e2e
```

Comment:
We have this naming convention currently: `pull-kubernetes-node-kubelet-serial-crio-cgroupv2`. Perhaps change this to `pull-kubernetes-node-kubelet-e2e-crio-imagefs-cgroupsv2`. Thoughts??

Reply:
@rphillips I think he has named it by following existing job like, https://github.com/kubernetes/test-infra/pull/31172/files#diff-9495ba672589316a9265fd090ecb9c3c74fd414a11386acafda27df4bd409409R980

Reply:
Followed up with @rphillips and we are okay with keeping names as is.